regression_and_census.R

```diff
@@ -24,36 +184,34 @@ doubleYScale(obj1, obj2, add.ylab2 = TRUE, use.style=FALSE )
 
 CA_PM25HOUR_plot_max <-
   ggplot(PM_LA, aes(week, PM25_wk_avg)) +
+  theme_bw() +
   geom_line(stat="identity")+ 
   ggtitle("Weekly Avg PM 2.5 in LA (AUG - DEC)")+
   labs(y="Level of PM 2.5 (Micrograms/cubic meter (LC))", x = "Date (week, Aug - Dec)")
 
 covid_LA <- ggplot(PM_LA, aes(week, ICU14lag_wkavg)) +
+  theme_bw() +
   geom_line(stat="identity")+ 
   ggtitle("Total number of people currently hospitalized in the ICU in LA")+
   labs(y="Total number of people currently hospitalized in the ICU", x = "Date (AUG - DEC)")
 
 #RAW   
-pdf("NO2_raw_CA_Apr1820.pdf") 
-grid.arrange(CA_PM25HOUR_plot_max, covid_LA, nrow = 2)
-dev.off()
+ggsave("NO2_raw_CA_Apr1820.pdf", arrangeGrob(CA_PM25HOUR_plot_max, covid_LA, nrow = 2),
```

Comment:
`ggsave` is a useful function for saving `ggplot` objects to pdf or png. You don't need to explicitly specify the file format, as long as you have the right extension (here .pdf).